cypress/support/commands.js

```diff
@@ -32,4 +32,12 @@ Cypress.Commands.add('typeAndSelectCke', (text) => {
   });
 });
 
-
+Cypress.Commands.add('addManifest', (title, manifest, timeout, wait) => {
+  cy.get('a[href="https://scalar-dev.fas.harvard.edu/features-test-book/new.edit?type=media&"]').click({force: true})
```

Comment:
Same question about this attribute selector and making it less dependent on the specific server that's being tested.